source/guides/clear/mixer.rst

```diff
@@ -411,27 +479,197 @@ mix version 10 (from which the image was built), to mix version 20.
+
+   .. code-block:: bash
+
+      mixer build all
+
+   .. note::
+      :command:`mixer build all` runs both :command:`mixer build bundles` and :command:`mixer build update` in one step.
+
+At this point the new mixer version is complete. All the content of the
+editors bundles is marked as deleted. If any clients of this mix want to
+upgrade to mixer build version 20, the editors bundle’s content will have
+been removed. Note that the bundle still exists and is being tracked by
+swupd, but it contains no files. To learn how to remove the bundle from
+being listed and tracked by swupd, see **the following example**.
+
+Example 7: Execute a format bump
+================================
+
+As a maintainer of your mix, you must execute a format bump if you wish to:
+* Track upstream’s format bump on your downstream derivative
+* Delete any custom bundles that were added
+
+Follow the appropriate use case below depending on your needs.
+
+Basic
+-----
+
+If you maintain your own downstream derivative and you want to track
+upstream, you need to do a format bump when one occurs on upstream. This
+method helps you track the latest changes on upstream; however, it does not
+change any local content that was added or deleted. For example, if you
+deprecated bundles, this method will **not remove the bundle tracking**.
+Refer to `Advanced`_ for help on managing your local mix and removing bundle
+tracking.
+
+In this example, we show a mix version that was initialized to upstream
+version 29740. You need to update your mix to upstream version 30700.
+To do so, you will go through a format bump.
+
+#. Change to your mix location, and verify the installed version, and the
+   format, of your mix.
+
+   .. code-block:: bash
+
+      mixer versions
+
+#. Update to upstream using the installed version from the previous step:
+
+   .. code-block:: bash
+
+      mixer versions update --upstream-version 30700
+
+   The output will look like this:
+
+   .. code-block:: console
+
+      Old mix:      10
+      Old upstream: 29740 (format: 27)
+
+      New mix:      20
+      New upstream: 30700 (format: 28)
+      [...]
+
+   Read the output carefully:
+
+   * The Old mix shows the current version (10) of  your mix.
+
+   * The Old upstream shows the version and format (27) on which it’s based.
+
+   * The New mix shows the new version (20) of your mix.
+
+   * The New upstream shows the version and format (28) on which it’s based.
+
+#. Given that the format in the output differs, you need to run a
+   format bump:
+
+   .. code-block:: bash
+
+       sudo mixer build upstream-format --new-format 28
+
+   .. note::
+
+      You specify the :command:`--new-format` to indicate the format (28) to which you transition.
+
+#. Your mix is now synchronized with the new format (28); however, you must
+   still advance to the desired or latest version.
+
+   .. code-block:: bash
+
+      mixer versions update --upstream-version 30700
```

Comment:
Do we have to do this step twice?  If so, then explain you have to run this command a 2nd time.

Reply:
@rchiossi. Can you remember why we added this? It's been 5 weeks since we discussed this.

Reply:
The explanation is right above it. `Your mix is now synchronized with the new format (28); however, you must still advance to the desired or latest version.`
The first time you ran it, if didn't succeed because there was a format bump on the way. You solved the format bump problem, now you run the command again.

Reply:
Yes. I remember testing it that way. LGTM.